module/zfs/dmu.c

```diff
@@ -1767,7 +1770,7 @@ dmu_write_policy(objset_t *os, dnode_t *dn, int level, int wp, zio_prop_t *zp)
 		 * pipeline.
 		 */
 		compress = ZIO_COMPRESS_OFF;
-		checksum = ZIO_CHECKSUM_OFF;
+		checksum = ZIO_CHECKSUM_NOPARITY;
```

Comment:
Will this break backwards compatibility with anything if the ZIO_CHECKSUM_OFF enum is recorded anywhere?